__tests__/server.tests.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
+const axios = require('axios');
+// const getOne = require('./server/models.js');
+// describe('server tests', () => {
+test('a get request should return an object', async () => {
+  const findOneObject = await axios.get('http://127.0.0.1:3000/');
```

Comment:
Would recommend using something like `supertest` here so you do not have to rely on the developer spinning up the server on their own first.